docs/pb_wgs.md

```
@@ -0,0 +1 @@
+#TODO
```

markdownlint-cli2 (0.22.0)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)